app/src/main/java/org/simple/clinic/drugs/EditMedicinesModel.kt

```diff
@@ -9,12 +9,13 @@ import java.util.UUID
 data class EditMedicinesModel(
     val patientUuid: UUID,
     val prescribedDrugs: List<PrescribedDrug>?,
-    val protocolDrugs: List<ProtocolDrugAndDosages>?
+    val protocolDrugs: List<ProtocolDrugAndDosages>?,
+    val openIntention: OpenIntention?
```

Comment:
Also tracking open intention in `EditMedicinesModel` should have been a separate commit.